src/main/java/it/pagopa/pm/gateway/dto/vpos/VposResumeMethodResponse.java

```diff
@@ -0,0 +1,13 @@
+package it.pagopa.pm.gateway.dto.vpos;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class VposResumeMethodResponse {
+
+    String requestId;
```

Comment:
Given that this class handles UUID only, change to:
```suggestion
    UUID requestId;
```

Reply:
https://github.com/pagopa/pagopa-payment-transactions-gateway/pull/200/commits/97e806ae8f4d63c8281711d220999374caf7024b